ipynb/FilterAndPickOTUs.ipynb

```diff
@@ -0,0 +1,329 @@
+      "\n",
+      "* Filter demultiplexed sequence data to only fecal samples\n",
+      "* Determine what sequences in the fecal samples recruit to the observed bloom sequences\n",
+      "* Remove the recruited bloom sequences from the demultiplexed sequence data\n",
+      "* Trim the filtered demultiplexed data (to reduce study bias when combining with short reads, such as the data in [Yatsunenko et al 2012](http://www.nature.com/nature/journal/v486/n7402/abs/nature11053.html))\n",
+      "* Pick OTUs against Greengenes 13_8 for the trimmed and full length filtered demultiplexed data\n",
+      "\n",
+      "The notebook is designed to support bloom filtering multiple datasets. \n",
+      "\n",
+      "The filtering is only intended to be applied to fecal data. As such, this notebook allows you to describe what metadata column and value to use so that only fecal samples are used."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "%run cluster_utils.ipy\n",
+      "import os\n",
+      "from functools import partial\n",
+      "\n",
+      "current_home = os.environ['HOME']\n",
+      "basedir = os.path.join(current_home, \"ICU\")\n",
+      "working_dir = os.path.join(basedir, 'blooming')\n",
+      "prj_name = 'blooming'\n",
+      "trim_length = 100\n",
+      "\n",
+      "# define a submit function against qsub\n",
+      "def submit(cmd_prefix, prj_name, queue, extra_args, cmd):\n",
+      "    \"\"\"Submission wrapper to submit to qsub\"\"\"\n",
+      "    cmd = \"%s; %s\" % (cmd_prefix, cmd)\n",
+      "    return submit_qsub(cmd, job_name=prj_name, queue=queue,\n",
+      "                       extra_args=extra_args)\n",
+      "\n",
+      "# define any environment specific details for any running compute job\n",
+      "cmd_prefix = 'source ~/.bash_profile; workon ag'\n",
+      "\n",
+      "# bootstrap the submit method\n",
+      "submit_8gb = partial(submit, cmd_prefix, prj_name, 'short8gb', '-l walltime=23:59:59')\n",
+      "submit_smr = partial(submit, cmd_prefix, prj_name, 'long8gb', '-l nodes=1:ppn=32 -l walltime=72:00:00')\n",
+      "\n",
+      "# setup a working directory\n",
+      "if not os.path.exists(working_dir):\n",
+      "    os.mkdir(working_dir)\n",
+      "path_joiner = lambda x: os.path.join(working_dir, x)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Here is where we'll setup paths for our input datasets"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "# setup the paths to your input datasets\n",
+      "paths_to_input_datasets = [\"/Users/mcdonadt/scratch/ICU/r1/seqs_r1.fna\",\n",
```

Comment:
These are for the user to change, and their data may not be under the basedir (as is the case here)